Class-10-(Scope-Exceptions-Types).md

```diff
@@ -0,0 +1,23 @@
+## In the prep work this week, the students learned:
+1. More about scope. Scope was covered briefly in the Functions chapter, however, now students have covered it more in-depth in terms of the order code executes in.
+1. What exceptions are.
+1. How to raise an exception.
+1. More about what a primitive data type is.
+
+## Announcements
+1. More info to come... when in doubt, check with your class Candidate Engagement Manager!
+
+## Large Group Time (Instructor)
+
+### Class 10 Topics That Require Careful Attention
+1. Conversation starters:
+   1. What is the difference between different types of errors.
+   1. What are some more common errors?
+   1. How do we raise an exception?
+   1. How do we handle exceptions?
+
+## Studio (TA Notes)
+
+### Class 10 Studio
+1. Don't be afraid to share your own experience!
+1. This studio is aimed on helping people develop processes for more strategically debugging their code. If after this studio, they are still struggling with that, don't hesitate to gently remind them of the topics covered here.
```

Comment:
Rephrasing suggestion for the second sentence: "After this studio, if they are still struggling, do not hesitate to gently remind them of the topics covered here."
OR 
"If they still struggle with debugging after this studio, gently remind them of the topics covered here."

Reply:
Done! Thanks!